packages/cli/src/build.ts

```diff
@@ -145,40 +158,38 @@ export function buildHandler(options: BuildHandlerOptions) {
   const files = getFiles(customFiles, tsConfig.fileNames);
 
   const compilerOptions = getCompilerOptions(baseOptions);
-  const program = getProgram({ compilerOptions, files, host });
+  const program = getProgram({
+    compilerOptions,
+    files,
+    host,
+    projectReferences: tsConfig.projectReferences,
```

Comment:
What happens if the tsconfig contains references but we aren't building with `--references`?

Reply:
Then the project is built using the normal logic, which likely doesn't do much depending on the config. `--references` is enabled by default if the project has references:

https://github.com/ts-bridge/ts-bridge/blob/8f4af8c72ec7e35f24fde60202024f37dc61ba27/packages/cli/src/build.ts#L392-L394

Reply:
Yeah, I meant does adding `projectReferences` to `getProgram` alter the compilation in any way without `--references`? I know that we don't do anything differently if the CLI parameter isn't present, but I was curious if TS potentially would.

Reply:
I doesn't seem to do anything differently.